### PR TITLE
[CPDLP-2431] Appropriate body dashboard view updates

### PR DESCRIPTION
**Pull request by mooktakim**
### Context

- Ticket: https://dfedigital.atlassian.net/browse/CPDLP-2431

### Changes proposed in this pull request

* Updated AB participants page to new table layout
* Updated CSV export to match new table
* Updated status name and description to excel matrix

### CODE REVIEW

File: app/services/appropriate_bodies/participants_filter.rb [hunk 1]
```diff
@@ -49,23 +43,6 @@ def filter_query(scoped, query)
       ).ransack("#{fields}_cont": query).result.distinct
```
Comment (cwrw) [inline app/services/appropriate_bodies/participants_filter.rb:43]: do we need to cut down some of those fields or should we keep them all?

File: app/views/appropriate_bodies/participants/index.html.erb [hunk 2]
```diff
@@ -10,17 +12,6 @@
   title: "Search",
   hint: "For example, school, participant name or teacher reference number (TRN)",
```
Comment (cwrw) [inline app/views/appropriate_bodies/participants/index.html.erb:13]: could you update the content here to match the prototype?

File: app/views/appropriate_bodies/participants/index.html.erb [hunk 3]
```diff
@@ -1,6 +1,8 @@
 <span class="govuk-caption-xl">Appropriate body</span>
```
Comment (cwrw) [inline app/views/appropriate_bodies/participants/index.html.erb:1]: I'm not sure we need this anymore?

File: app/views/appropriate_bodies/participants/index.html.erb [hunk 4]
```diff
@@ -34,16 +25,11 @@
...
-      <th scope="col" class="govuk-table__header">Role</th>
-      <th scope="col" class="govuk-table__header">Lead provider</th>
-      <th scope="col" class="govuk-table__header">Delivery partner</th>
-      <th scope="col" class="govuk-table__header">School</th>
-      <th scope="col" class="govuk-table__header">School unique reference number</th>
-      <th scope="col" class="govuk-table__header">Academic year</th>
-      <th scope="col" class="govuk-table__header">Training status</th>
-      <th scope="col" class="govuk-table__header">Status</th>
+      <th scope="col" class="govuk-table__header">School URN</th>
+      <th scope="col" class="govuk-table__header">Status (TBC)</th>
```
Comment (cwrw) [inline app/views/appropriate_bodies/participants/index.html.erb:30]: not sure we need the TBC?

File: spec/components/appropriate_bodies/participants/table_row_spec.rb [hunk 5]
```diff
@@ -1,29 +1,20 @@
 # frozen_string_literal: true
 
 RSpec.describe AppropriateBodies::Participants::TableRow, type: :component do
-  let(:school_cohort) { create(:school_cohort) }
-  let!(:induction_record) { create :induction_record, participant_profile: }
+  let(:participant_profile) { create :ect_participant_profile, training_status: "withdrawn" }
```
Comment (cwrw) [inline spec/components/appropriate_bodies/participants/table_row_spec.rb:4]: could you add a test for CIP?

File: app/services/appropriate_bodies/participants_filter.rb [hunk 6]
```diff
@@ -10,20 +10,14 @@ def initialize(collection:, params:)
     end
 
     def scope
-      scoped = collection
+      scoped = collection.includes(
```
Comment (cwrw) [inline app/services/appropriate_bodies/participants_filter.rb:13]: could we add a spec for this service?

**Review by cwrw — COMMENTED** [review]
Looking good a few comments. The performance as we've known isn't great until we fix the status issue, anything else regarding n+1 we can find and improve in the serialiser would be great as well

File: app/services/appropriate_bodies/participants_filter.rb (hunk 1 above)
Comment (mooktakim) [reply app/services/appropriate_bodies/participants_filter.rb:43]: Sure, will look to reduce the fields

File: app/views/appropriate_bodies/participants/index.html.erb (hunk 2 above)
Comment (mooktakim) [reply app/views/appropriate_bodies/participants/index.html.erb:13]: Updated

File: app/views/appropriate_bodies/participants/index.html.erb (hunk 3 above)
Comment (mooktakim) [reply app/views/appropriate_bodies/participants/index.html.erb:1]: I was thinking about this. There isn't any way to know you're in the AB section. We have a similar caption for DP. I'm not sure if we should remove this.

File: app/views/appropriate_bodies/participants/index.html.erb (hunk 4 above)
Comment (mooktakim) [reply app/views/appropriate_bodies/participants/index.html.erb:30]: Updated

File: spec/components/appropriate_bodies/participants/table_row_spec.rb (hunk 5 above)
Comment (mooktakim) [reply spec/components/appropriate_bodies/participants/table_row_spec.rb:4]: Added

File: app/services/appropriate_bodies/participants_filter.rb (hunk 6 above)
Comment (mooktakim) [reply app/services/appropriate_bodies/participants_filter.rb:13]: Added

**Review by cwrw — APPROVED** [review]
🙌

**Pull request merged**